modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/npm/JSBundleAsset.java

```diff
@@ -18,12 +18,26 @@
 import java.io.InputStream;
 
 /**
+ * A {@link JSBundleObject} which contents can be requested from the outside
+ * world using a portal URL or by means of an {@link InputStream}.
```

Comment:
@izaera Does this class description accurately describe this class:

> Provides contents of an OSGi bundle containing NPM packages and modules. These contents can be requested from external apps by using a portal URL or {@link InputStream}.

Let me know if it needs and edits or additions.